src/main/java/org/cloudburstmc/server/player/CloudPlayer.java

```diff
@@ -1564,7 +1564,7 @@ public void completeLoginSequence() {
         startGamePacket.setBroadcastingToLan(true);
         NetworkUtils.gameRulesToNetwork(this.getLevel().getGameRules(), startGamePacket.getGamerules());
         startGamePacket.setLevelId(""); // This is irrelevant since we have multiple levels
-        startGamePacket.setLevelName(this.getServer().getNetwork().getName()); // We might as well use the MOTD instead of the default level name
+        startGamePacket.setLevelName(this.getServer().getNetwork().getMotd()); // We might as well use the MOTD instead of the default level name
```

Comment:
I believe @SupremeMortal wanted to have the Level name so that on the player list screen it shows the level name at the top instead of a MOTD that you'd only see 8 characters of.

Reply:
but the getName method only returns the name field of the Network object, which in turn is set to MOTD during the CloudServer's boot process, and that is the only place in the codebase that sets the value of the field.

Maybe we should come with a better name other than MOTD and subMOTD, or is this just a bug buried in the code?